content/blog/17-18/vleor.md

```diff
@@ -0,0 +1,108 @@
+---
+author: Rien Maertens
+title: "Het avontuur van de Vleor"
```

Comment:
Ik zou eerlijk gezegd er een [(sic)](https://www.wikiwand.com/en/Sic) achter zetten, en mss een klein paragraafje toevoegen (mss rond 2017 toen die dude bij ons kwam en iedereen VLEOR VOELR VLOER spamde) waarom we het zo spellen

Reply:
Zou nu duidelijker moeten zijn. Ik weet wel niet of ik fan ben van de sic in de titel.

Reply:
Met de uitleg is het al een pak beter. De sic zou ik er zetten zodat we geen mails krijgen van "tis verkeerd gespeld", maar met een uitleg is het al een pak beter 👍

Reply:
Oh zeker met die dictionary definition, ik ben fan!